src/servers/CMakeLists.txt

```diff
@@ -203,13 +203,15 @@ set(
   common.cc
   shared_memory_manager.cc
   ../core/logging.cc
+  ../core/model_config.cc
```

Comment:
To HTTP_ENDPOINT_OBJECTS above can you add model-config-library instead of doing this.

Reply:
Done